Dockerfile

```diff
@@ -21,7 +24,7 @@ RUN chmod +x /usr/local/bin/supw
 ARG BZIP_URL=https://github.com/mumble-voip/mumble/releases/download/${MUMBLE_VERSION}/murmur-static_x86-${MUMBLE_VERSION}.tar.bz2
 
 # Install dependencies, fetch Mumble bzip archive and chown files
-RUN apk add --update ca-certificates bzip2 tar tzdata wget \
+RUN apk add --update ca-certificates su-exec bzip2 gettext tar tzdata wget \
```

Comment:
Is `gettext` still needed?

Reply:
`gettext` is used to do variable substitution in strings using `envsubst`. If the last line of the entrypoint script is hardcoded, then we do not need it.